pkg/audit/verifier.go

```diff
@@ -369,9 +374,24 @@ func (verifier *Verifier) Reverify(ctx context.Context, stripe *Stripe) (report
 				}
 
 				if errs2.IsRPC(err, codes.NotFound) {
+
+					// Get the original segment pointer in the metainfo
+					oldPtr, err := verifier.metainfo.Get(ctx, pending.Path)
+					remoteSegment := oldPtr.GetRemote()
+					if err != nil || remoteSegment == nil || remoteSegment.RootPieceId != pending.PieceID {
+						ch <- result{nodeID: piece.NodeId, status: skipped}
```

Comment:
I think we should report `success` status here. The node should exit the containment mode as we cannot reverify it -- the segment it didn't respond to an audit is gone now.